src/start.cli.ts

```diff
@@ -109,3 +139,7 @@ function readVersion(): string {
         return "@atomist/automation-client 0.0.0";
     }
 }
+
+/* Local Variables:  */
+/* mode: typescript  */
+/* End:              */
```

Comment:
What’s that?

Reply:
Emacs see the node line at the top of the file and wants to use JavaScript mode. This was the easiest way to overrides that. I need to find a better way to address that, so I can remove it if you want.

Reply:
Yeah, would be. Ice to remove it. Feels like it doesn’t belong there.